src/components/box/box.stories.mdx

```diff
@@ -0,0 +1,66 @@
+import { Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+import Box from '.';
+import Form from '../form';
+import Textbox from '../../__experimental__/components/textbox';
+import Button from '../button';
+import StyledSystemProps from '../../../.storybook/utils/styled-system-props';
+
+<Meta title="Box" />
+
+# Box
+
+```javascript
+  import Box from 'carbon-react/lib/components/box';
+```
+
+## Box
+Basic component that gives access to a number of styling options. Should be used to achieve more precise layouts when required. Gives access to Spacing, Color, Layout and FlexBox attributes
+<Preview>
+  <Story name="default" parameters={{ info: { disable: true }}} >
+    <>
```

Comment:
I wouldn't expect a fragment to appear in the Storybook code sample, do we need it here?

Reply:
Not needed and removed